pyaerocom/io/mscw_ctm/reader.py

```diff
@@ -505,13 +554,27 @@ def _open_file(self):
         """
         fps = self._filepaths
         ds = {}
-
         yrs = self._get_yrs_from_filepaths()
 
-        ts_type = self._ts_type_from_filename(self._filename)
+        ts_type = self._ts_type
         fps = self._clean_filepaths(fps, yrs, ts_type)
-        if len(fps) > 1 and ts_type == "hourly":
-            raise ValueError(f"ts_type {ts_type} can not be hourly when using multiple years")
+
+        if ts_type == "hourly":
+            years = set()
+            for fp in self._private.files:
+                with xr.open_dataset(fp) as nc:
+                    start_year = (
+                        nc["time"][:].data.min().astype("datetime64[Y]").astype(int) + 1970
+                    )
+                    end_year = nc["time"][:].data.max().astype("datetime64[Y]").astype(int) + 1970
+                for y in range(start_year, end_year + 1):
+                    years.add(y)
```

Comment:
What about a seasonal run for winter, e.g. (Dec,Jan,Feb), which is strictly not multi-year? Rather check if end-date - start-date in hours > 366*24 - we just want to avoid too many datapoints.